### DISCUSSION TITLE
"Save as" feature when I need to save a file locally

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#224] [General]
Currently there is only "Share as" feature, but I want saving a file directly in a directory of my device.

**Comment by aritra-tech** [comment]
> Currently there is only "Share as" feature, but I want saving a file directly in a directory of my device.

Yeah, actually I was thinking of a directory saving along with tags.

**Reply by aritra-tech** [reply]
@serrq  please mark this as an answer if you have no further queries so that others can know that the question has been answered.

**Reply by serrq** [reply]
Will this become a work? If yes, move it into issues.

If no, I close it as not planned.
